src/components/modal.rs

```diff
@@ -45,6 +45,10 @@ pub struct ModalProperties {
     #[prop_or_default]
     pub onclose: Option<Callback<()>>,
 
+    /// Disable close button
+    #[prop_or_default]
+    pub disable_close_button: bool,
```

Comment:
I would prefer this was called `show_close` and defaulted to `true` so that it ties up with the [PaternFly React docs](https://patternfly-react-v5.surge.sh/components/modal#props).